docs/src/main/paradox/kafka.md

```diff
@@ -1,5 +1,23 @@
 # Messages from Kafka
 
+A typical source for Projections is messages from Kafka. Akka Projections has integration with 
+[Alpakka Kafka](https://doc.akka.io/docs/alpakka-kafka/current/), which is described in here.
+
+The @apiddoc[KafkaSourceProvider] uses consumer group assignments from Kafka and can resume from offsets stored in
+a database.
+
+Akka Projections can store the offsets from Kafka in a @ref:[relational DB with Slick](slick.md).
```

Comment:
While writing the docs for the testkit, I realised that this is problematic. 

Maybe better to use the Cassandra example so the Java and the Scala docs align.

Reply:
Good point. We must use Slick for the consuming and that's why I continued with Slick here. We can change that later though, I'm not planning to write the Java sample today.